includes/admin/class-wc-payments-admin.php

```diff
@@ -1135,9 +1136,14 @@ public function add_transactions_notification_badge() {
 	 * @return int The number of disputes which need a response.
 	 */
 	private function get_disputes_awaiting_response_count() {
+		$send_callback = function() {
+			$request = Request::get( WC_Payments_API_Client::DISPUTES_API . '/status_counts' );
+			return $request->send( 'wcpay_get_dispute_status_counts' );
```

Comment:
Sorry, I would have caught it up earlier. I've manaully tested this again, and it requires an array as the output instead of an object: 

```suggestion
			return $request->send( 'wcpay_get_dispute_status_counts' )->to_array();
```